cmd/storer2elastic/dataprocessor/dataReplayer.go

```diff
@@ -0,0 +1,583 @@
+package dataprocessor
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	storer2ElasticData "github.com/ElrondNetwork/elrond-go/cmd/storer2elastic/data"
+	"github.com/ElrondNetwork/elrond-go/cmd/storer2elastic/databasereader"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/core/indexer"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/data/rewardTx"
+	"github.com/ElrondNetwork/elrond-go/data/smartContractResult"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/data/typeConverters"
+	"github.com/ElrondNetwork/elrond-go/hashing"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/ElrondNetwork/elrond-go/storage"
+)
+
+type dataReplayer struct {
+	databaseReader   DatabaseReaderHandler
+	shardCoordinator sharding.Coordinator
```

Comment:
from the shard coordinator just the numShards is used.

Reply:
yes, but I would leave it like this

Reply:
ok